src/sf/grid/social-media/posts.twitter.suma

```diff
@@ -0,0 +1,115 @@
+// Twitter API Reference: https://developer.twitter.com/en/docs/twitter-api
+
+profile = "social-media/posts@1.0"
+provider = "twitter"
+
+map GetProfilePosts {
+
+  http GET "/2/users/{input.profileId}/tweets" {
+
+    request {
+      query {
+        'tweet.fields' = 'id,text,created_at,attachments'
+        expansions = 'attachments.media_keys'
+        'media.fields' = 'duration_ms,height,media_key,preview_image_url,type,url,width,alt_text'
+        start_time = input.afterDate
+        end_time = input.beforeDate
+        pagination_token = input.page
+        max_results = 25
+      }
+      headers {
+        authorization = "Bearer " + parameters.accessToken
+      }
+    }
+
+    response 200 "application/json" {
+      posts = []
+
+      set if (body.meta.result_count > 0) {
+        //workaround map interpreter variables merging issue
+        posts = call foreach(index of Array(body.data.length - 1).fill().map((_, i) => i)) MapTweetToPost(
```

Comment:
@jnv this workaround prevents leaking of attachments from first tweet to next tweets on the page. It is caused by variables merging in map interpreter. I am going to report the issue.

Reply:
Yuck! I suppose using `return map result` won't fix that?

Reply:
nope, it did not help